portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -129,7 +150,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       long timestamp = System.currentTimeMillis();
 
       Entity blogMessageEntity = new Entity("blogMessage");
-      blogMessageEntity.setProperty("sender", sender);
+      blogMessageEntity.setProperty("nickname", sender);
```

Comment:
Not an urgent thing, but I also think we should update instances of sender through out the other files so that it's not confusing. Sometimes in data store there are separate columns for sender and for nickname when there should just be one.